src/common-helpers.ts

```diff
@@ -11,6 +11,7 @@ import { AuxMethod, Method } from "./method";
 import { contentTypes } from "./content-type";
 
 export type FlatObject = Record<string, unknown>;
+export type EmptyObject = Record<string, never>;
```

Comment:
this type does not allow assignments, but it works for reading when merged with another interface. It should not be breaking, but still addressing that change to v20, because it narrows some other publicly exposed interfaces

Reply:
addressed